### ISSUE TITLE
Please use FontUIResource instead of plain Font when configuring defaults

### ISSUE THREAD

**Issue by vlsi** [weisJ/darklaf#101]
See `javax.swing.LookAndFeel#installColorsAndFont`

E.g. `BasicLabelUI` sets the font with

```java
LookAndFeel.installColorsAndFont(c, "Label.background", "Label.foreground", "Label.font");
```

However, Darklaf sets the font as `new Font`, thus it prevents the font from being updated by LaF change.

---

Testcase: launch JMeter as `./gradlew runGui -PlocalDarklaf`, and click `options` - `zoom in/out`.

**Comment by weisJ** [comment]
Ah this issue only occurs on macOS due to the code that replaces the fonts with the system fonts. Generally every font that the laf (or the `PropertyLoader` for that matter) creates is already a `FontUIResource`.

**Comment by weisJ** [comment]
Fixed in 7c760cf759aec60dee1dbad445e03b4359636534
